src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-password-form/user-password-form.component.html

```diff
@@ -0,0 +1,117 @@
+          </div>
+        </div>
+
+        <!-- New password -->
+        <div class="form-group"
+             [ngClass]="{'has-error': userForm.showError('newpassword', frm)}">
+          <label class="control-label col-sm-3"
+                 for="newpassword">
+            <ng-container i18n>New password</ng-container>
+            <span class="required"></span>
+          </label>
+          <div class="col-sm-9">
+            <div class="input-group">
+              <input class="form-control"
+                     type="password"
+                     placeholder="Password..."
+                     id="newpassword"
+                     autocomplete="new-password"
+                     formControlName="newpassword">
+              <span class="input-group-btn">
+                <button type="button"
+                        class="btn btn-default"
+                        cdPasswordButton="newpassword">
+                </button>
+              </span>
+            </div>
+            <span class="help-block"
+                  *ngIf="userForm.showError('newpassword', frm, 'required')"
+                  i18n>This field is required.</span>
+          </div>
+        </div>
+
+        <!-- Confirm new password -->
+        <div class="form-group"
+             [ngClass]="{'has-error': userForm.showError('confirmnewpassword', frm)}">
+          <label class="control-label col-sm-3"
+                 for="confirmnewpassword">
+            <ng-container i18n>Confirm new password</ng-container>
+            <span class="required"></span>
+          </label>
+          <div class="col-sm-9">
+            <div class="input-group">
+              <input class="form-control"
+                     type="password"
+                     autocomplete="off"
+                     placeholder="Confirm new password..."
+                     id="confirmnewpassword"
+                     formControlName="confirmnewpassword">
+              <span class="input-group-btn">
+                <button type="button"
+                        class="btn btn-default"
+                        cdPasswordButton="confirmnewpassword">
+                </button>
+              </span>
+            </div>
+            <span class="help-block"
+                  *ngIf="userForm.showError('confirmnewpassword', frm, 'required')"
+                  i18n>This field is required.</span>
+            <span class="help-block"
+                  *ngIf="userForm.showError('confirmnewpassword', frm, 'match')"
+                  i18n>Password confirmation doesn't match the new password.</span>
+          </div>
+        </div>
+      </div>
+
+      <div class="panel-footer">
+        <div class="button-group text-right">
+          <cd-submit-button (submitAction)="onSubmit()"
+                            [form]="userForm"
+                            i18n="form action button|Example: Create Pool@@formActionButton"
+                            type="button">
```

Comment:
This is not needed and will cause visual problems in some browsers.
```suggestion
>
```